shop/src/Components/Card/index.jsx

```diff
@@ -1,26 +1,29 @@
 import { NavLink } from "react-router-dom";
-import "./Card.css";
-const IMG_CARD =
-  "https://demo.evershop.io/assets/homepage/banner/men-shoes.jpeg";
-const Card = () => {
+import "./Card.scss";
+const Card = ({ imgAndPathOfHomeCard }) => {
   return (
-    <div className="cardContainer">
-      <img src={IMG_CARD} alt="" />
+    <>
+      {imgAndPathOfHomeCard.map((content) => {
```

Comment:
if `imgAndPathOfHomeCard` is empty, this component will break. So please add some sort of fallback of the `imgAndPathOfHomeCard` is empty